lib/index.js

```diff
@@ -125,27 +123,56 @@ function junkFoodMove(junkFood){
  junkFood.y += junkFood.speed;
 }
 
-function isHealthFoodCollidingWithTray(healthFood, tray){
-    return !(
-      tray.x > healthFood.x + healthFood.width ||
-      tray.x + tray.width < healthFood.x ||
-      tray.y > healthFood.y + healthFood.height ||
-      tray.y + tray.height < healthFood.y);
-}
+window.setInterval(collisionOutcome, 200, healthFoods, world.tray);
 
 function collisionOutcome(healthFoods, tray){
-  for (var i = 0; i < healthFoods.length; i++)
-  {var healthFood = healthFoods[i];
+  for (var i = 0; i < healthFoods.length; i++){
+    var healthFood = healthFoods[i];
     if (isHealthFoodCollidingWithTray(healthFood, tray)){
       recordHealthFoodScore(healthFood);
     }
   }
 }
 
+function isHealthFoodCollidingWithTray(healthFood, tray){
+  return !(
+    tray.x > healthFood.x + healthFood.width ||
+    tray.x + tray.width < healthFood.x ||
+    tray.y > healthFood.y + healthFood.height ||
+    tray.y + tray.height < healthFood.y
+  );
+}
+
 function recordHealthFoodScore(healthFood){
   var newScore = +healthFood.score;
   healthFoods.shift(healthFood);
   total += newScore;
   document.getElementById('score-game').innerHTML = "Score: " + total;
 }
-window.setInterval(collisionOutcome, 200, healthFoods, world.tray);
+
+window.setInterval(junkCollisionOutcome, 200, junkFoods, world.tray);
+
+function junkCollisionOutcome(junkFoods, tray){
+  for (var i = 0; i < junkFoods.length; i++){
+    var junkFood = junkFoods[i];
+    if (isJunkFoodCollidingWithTray(junkFood, tray)){
+      decreaseScore(junkFood);
+    }
+  }
+}
+
+function isJunkFoodCollidingWithTray(junkFood, tray){
+  return !(
```

Comment:
I think we could refactor this with the healthFood one again to save ourselves from repetitive code. Looks great!